### ISSUE TITLE
partial_tempering.sh only bond parameters

### ISSUE THREAD

**Issue by MauriceKarrenbrock** [plumed/plumed2#794]
Hello,

partial_tempering.sh if I'm not wrong does scale both the bond parameters and the non bond ones (VdW and Q), so I was wondering if it would be possible to add a flag in partial_tempering.sh in order to choose to scale only the bond parameters (torsions, vibrations, bendings, etc) without touching the van der waals and coulombic interactions?
Something like
`plumed partial_tempering --only-bound ...`

Thank you very much in advance!

**Comment by GiovanniBussi** [comment]
@MauriceKarrenbrock I think you can try to obtain that effect by editing the partial_tempering awk script (I guess you should replace all appearance of `scale` in [this region](https://github.com/plumed/plumed2/blob/v2.8.0/scripts/partial_tempering.sh#L277-L312) with `1.0`). Anyway notice that:
- bonds and bends are not touched by my script, only torsions
- modifying torsions without modifying the corresponding charges and VdW will introduce an inconsistency between torsions and 1-4 interactions

Due to the second point, I am a bit skeptical in including this as an option. Tough I see it might be interesting to play with it (and even allow different factors for charge and LJ).

**Comment by MauriceKarrenbrock** [comment]
Hello,

Thank you very much for the suggestion on how to modify the script!

> bonds and bends are not touched by my script, only torsions
Even better!

> modifying torsions without modifying the corresponding charges and VdW will introduce an inconsistency between torsions and 1-4 interactions
What do you mean, do you expect some strange conformations to become more stable than they should?

In any case when I have some extra time I will do some tests to see what happens and will keep you updated

**Comment by GiovanniBussi** [comment]
>What do you mean, do you expect some strange conformations to become more stable than they should?

Typically when you decrease the torsional potential you do so with the assumption that this will lower barriers. However, barriers are also affected by the 1-4 charge and LJ interactions. However, this is mostly intuition, I might be wrong (and the answer could be system dependent).

**Comment by MauriceKarrenbrock** [comment]
Hi @GiovanniBussi 

Even though this issue has been closed for a long time I wanted to let you know that I came up with a more flexible implementation of the `partial_tempering.sh` script.
It uses `Parmed` and allows you to independently and arbitrarily scale the torsions, VdW, Q, and 1-4 interactions of any set of atoms of the system.

I have only tried it on `Gromacs` topologies, and in that case it does produce correct output topology files, but on paper it should work on any format supported by `Parmed`.

The downsides are that this implementation is way slower than `partial_tempering.sh`, and that there is a bug in older `Gromacs` versions when explicitly defining `[ pairs ]` that makes it impossible to use the topologies where 1-4 interactions have been scaled selectively  [Gromacs issue #4769](https://gitlab.com/gromacs/gromacs/-/issues/4769)

I thought you might find it useful, and in case you happen to need it in the future the code can be found at the link below and is open-source

https://github.com/MauriceKarrenbrock/HREMGromacs/blob/9cc3381fb6f7f67ca911a60a288cb9423d0bbcd3/HREMGromacs/solute_tempering.py#L532-L618
